server/matchmaker/matchmaker_queue.py

```diff
@@ -247,6 +247,9 @@ def to_dict(self):
             "queue_pop_time": datetime.fromtimestamp(
                 self.timer.next_queue_pop, timezone.utc
             ).isoformat(),
+            "queue_pop_time_delta": timedelta(
+                seconds=self.timer.next_queue_pop - time.time()
+            ).total_seconds(),
```

Comment:
Is there really a point to converting from seconds to `timedelta` and back again here? Just seems funny to use the `seconds` parameter and then call `total_seconds` :P

Reply:
`self.timer.next_queue_pop` would be seconds already, yeah, but seconds since epoch plus some small scheduling window (`QUEUE_POP_TIME_MAX`). I assumed you wanted the seconds remaining until the next queue pop, but maybe you actually wanted to return a `timedelta` object?

Reply:
What I mean is this code is basically doing this:
```python3
assert timedelta(seconds=30).to_seconds() == 30
```
Converting the number of seconds to a timedelta and then back to the original number. Wouldn't it be simpler just to use
```python3
"queue_pop_time_delta": self.timer.next_queue_pop - time.time()
```

Reply:
Ah gotcha, ok!